gnomad_constraint/pipeline/constraint_pipeline.py

```diff
@@ -251,6 +254,8 @@ def main(args):
     custom_vep_annotation = args.custom_vep_annotation
     gerp_lower_cutoff = args.gerp_lower_cutoff
     gerp_upper_cutoff = args.gerp_upper_cutoff
+    coverage_metric = args.coverage_metric
```

Comment:
Make sure that these are added as globals somewhere, ignore if this is done already below